lib/identity_cache.rb

```diff
@@ -55,6 +56,9 @@ class << self
     mattr_accessor :fetch_read_only_records
     self.fetch_read_only_records = true
 
+    mattr_accessor :blob_replication_factor
+    self.blob_replication_factor = 1
```

Comment:
I'm wondering if we should be able to do this per-call, and not just per-model? Do we actually see ourselves _not_ turning this on everywhere? On any path that uses IDC, shouldn't we exactly spread it across multiple servers?